src/modules/helpers.js

```diff
@@ -0,0 +1,12 @@
+class HelperFuncs {
```

Comment:
Try to add comments to your code to help guide the reviewer on what you are doing.